packages/next/src/server/web/spec-extension/unstable-cache.ts

```diff
@@ -341,22 +337,17 @@ export function unstable_cache<T extends Callback>(
         // to allow tracking which "mode" we are in without the presence of a store or not. For now I have
         // maintained the existing behavior to limit the impact of the current refactor
         const result = await workAsyncStorage.run(
-          // We are making a fake store that is useful for scoping fetchCache: 'force-no-store' and isUnstableCacheCallback: true
+          // We are making a fake store that is useful for scoping fetchCache: 'force-no-store'
           // The fact that we need to construct this kind of fake store indicates the code is not factored correctly
           // @TODO refactor to not require this fake store object
           {
-            // force any nested fetches to bypass cache so they revalidate
-            // when the unstable_cache call is revalidated
-            fetchCache: 'force-no-store',
-            isUnstableCacheCallback: true,
             route: '/',
             page: '/',
             isStaticGeneration: false,
             fallbackRouteParams: null,
             buildId: '', // Since this is a fake one it can't "use cache" anyway.
           },
```

Comment:
I leave this to avoid risk changes to Pages Router but we should probably try removing this now.

Reply:
I don't think we ever fully made this work in pages router so I think could be removed